SalesforceCommerceCloud/pwa-kit
# Generator Add Einstein to demo preset

# Description


Adds the Einstein configuration to projects generated using the App Generator with `retail-react-app-demo` preset.


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add Einstein configuration to the generator `retail-react-app-demo` preset.

# How to Test-Drive This PR

- Run:
```
GENERATOR_PRESET=retail-react-app-demo node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir /tmp/projectdemoeinstein
```
- Verify the `app/api.config.js` file includes the right values for `einsteinId` and `siteId`.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
